content/docs/faq/README.md

```diff
@@ -54,11 +54,15 @@ cert-manager publishes all events to the Kubernetes events mechanism, you can ge
 
 Due to the nature of the Kubernetes event mechanism these will be purged after a while. If you're using a dedicated logging system it might be able or is already also storing Kubernetes events.
 
-### What happens if a renewal doesn't happen? Will it be tried again after some time?
+### What happens if an issuance failed? Will it be retried?
```

Comment:
nitpick: I'd prefer "issuance fails"  rather than "an issuance failed" but I don't think it _needs_ to change!

nitpick: changing this title changes the anchor link too; do you think we should add a manual `<a>` so we preserve the old link, as I did in [this other PR](https://github.com/cert-manager/website/pull/1003/files#diff-87f6019faccfa562a228b696d2b5a8ff9c71392f66eb62db144f3fdf84c6b905R351-R353)? I don't feel strongly though! (the suggestion would require that https://github.com/cert-manager/website/pull/1003 be merged first since it uses the new `className`

```suggestion
### What happens if issuance fails? Will it be retried?

<a id="what-happens-if-a-renewal-doesnt-happen-will-it-be-tried-again-after-some-time" className="hidden-link"></a>
```